test/boulder-tools/build.sh

```diff
@@ -55,8 +55,9 @@ git clone https://github.com/SUNET/pkcs11-proxy && \
 
 # Setup SoftHSM
 echo "0:/var/lib/softhsm/slot0.db" > /etc/softhsm/softhsm.conf
-pkcs11-tool --module=/usr/lib/softhsm/libsofthsm.so --init-token --label token_label --so-pin 1234
-pkcs11-tool --module=/usr/lib/softhsm/libsofthsm.so --init-pin --label token_label --pin 5678 --login --so-pin 1234
+echo "1:/var/lib/softhsm/slot1.db" >> /etc/softhsm/softhsm.conf
+softhsm --slot 0 --init-token --label intermediate --pin 5678 --so-pin 1234
+softhsm --slot 1 --init-token --label root --pin 5678 --so-pin 1234
```

Comment:
The problem is that `test/make-softhsm.sh` both initializes the slots and fills them, while build.sh only initializes the slots, and waits for `test/entrypoint.sh` to fill them. We could change the model so build.sh both initializes and fills (possible using make-softhsm.sh), but that would make it a bit harder to swap out private keys in Docker mode.